### PR TITLE
Hotfix/make python2 compatible

### PR DESCRIPTION
**Pull request by llexical**

### CODE REVIEW

File: test.py [hunk 1]
```diff
@@ -38,7 +38,20 @@ def test_kafka_message_send(self):
         Test that no errors are thrown, and it doesn't hang when you send a message
         """
         producer = HerokuKafkaProducer(**kafka_params)
-        producer.send(TOPIC1, b"some message")
+        #producer.send(TOPIC1, b"some message")
```
Comment (rockse) [inline test.py]: I think this comment is not required anymore

**Review by rockse — APPROVED** [review]
<3

Just one comment

File: test.py (hunk 1 above)
Comment (llexical) [reply test.py]: lol xD good spot ahahha

**Review by rockse — APPROVED** [review]
Awesome!

**Comment by rockse** [comment]
Tests are failing on circleci

**Pull request merged**